adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/impl/MappedMessage.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.mqtt.impl;
+
+import org.eclipse.hono.adapter.mqtt.MqttContext;
+import org.eclipse.hono.util.ResourceIdentifier;
+
+import io.vertx.mqtt.messages.MqttPublishMessage;
+
+/**
+ * Structure to provide a mapped message.
+ */
+class MappedMessage {
+    private final MqttContext ctx;
+    private final ResourceIdentifier resource;
+    private final MqttPublishMessage message;
+
+    /**
+     * Creates a new mappedMessage.
+     * @param ctx The context used for this message
+     * @param resource The resourceIdentifier used for this message
+     * @param message The actual message with its payload
```

Comment:
I think you can do better than this.
Also, please insert an empty line between description and params.
Please finish the param description with a `.`